class/defaults.yml

```diff
@@ -2,4 +2,53 @@ parameters:
   vshn_sli_reporting:
     =_metadata:
       multi_tenant: true
-    namespace: syn-vshn-sli-reporting
+    namespace: ${_instance}
+    namespaceMetadata: {}
+
+    images:
+      sli_reporting:
+        registry: ghcr.io
+        repository: vshn/vshn-sli-reporting
+        tag: v0.1.2
+
+    auth:
+      user: admin
+      password: null
+
+    db_file: /data/data.db
+
+    server:
+      host: 0.0.0.0
+      port: 8080
```

Comment:
I don't think we should make internal config configurable through commodore. This makes it harder to make new component releases without technially breaking changes. I don't think we'll ever have a case where this needs to be changed.

Reply:
Example: We want to switch to kube-rbac-proxy. Which would be an amazing fit here.